### ISSUE TITLE
The question about dataset

### ISSUE THREAD

**Issue by qy-guo** [ghli16/GTAE-VF#2]
Thank you for your excellent work. Research on virulence factors is of great significance, and we appreciate that you have made the complete code and dataset available, which is highly commendable. We would like to kindly request the pre-split training, validation, and test sets as well, as this would greatly facilitate further research in this area.

**Comment by ghli16** [comment]
Thank you for your interest in our work. The dataset we used in our paper originates from deepVF. For the training and validation sets, we used the deepVF training set and further subdivided it. The test set follows that of deepVF. For more details, you can refer to the original work of deepVF.

**Comment by ghli16** [comment]
Thank you for your attention. We have uploaded the dataset in our latest work. Plsease set it at github. (https://github.com/ghli16/PLMVF).




At 2024-10-09 21:12:27, "pejoy0" ***@***.***> wrote:

Thank you for your excellent work. Research on virulence factors is of great significance, and we appreciate that you have made the complete code and dataset available, which is highly commendable. We would like to kindly request the pre-split training, validation, and test sets as well, as this would greatly facilitate further research in this area.
